### PR TITLE
CSS/properties: add box-align

### PR DESCRIPTION
**Pull request by caugner**
Source: https://developer.mozilla.org/en-US/docs/Web/CSS/box-align#Browser_compatibility

### CODE REVIEW

File: css/properties/box-align.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+{
+  "css": {
+    "properties": {
+      "box-align": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/box-align",
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "prefix": "-webkit"
```
Comment (caugner) [inline css/properties/box-align.json]: Should probably be `-webkit-`.

Comment (caugner) [reply css/properties/box-align.json]: Fixed in a07cde91.

File: css/properties/box-align.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                  {
+                    "type": "preference",
+                    "name": "layout.css.prefixes.webkit",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": true,
+                "prefix": "-moz"
```
Comment (Elchi3) [inline css/properties/box-align.json]: -moz-

File: css/properties/box-align.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            "opera": {
+              "version_added": true,
+              "prefix": "-webkit-"
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": [
+              {
+                "version_added": "5.1",
```
Comment (Elchi3) [inline css/properties/box-align.json]: 3.0 ?

File: css/properties/box-align.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                "prefix": "-webkit-"
+              },
+              {
+                "version_added": "1.1",
+                "version_removed": true,
+                "prefix": "-khtml"
```
Comment (Elchi3) [inline css/properties/box-align.json]: -khtml-

**Pull request merged**